### PR TITLE
Mueritz now hosting computed icvpn. and 207.10.in-addr.arpa.zone. zones

### PR DESCRIPTION
**Pull request by wusel42**
We create an icvpn. and 207.10.in-addr.arpa.zone. zone hourly now. IP addresses suck ;)

### CODE REVIEW

**Comment by wusel42** [comment]
Well; don't think I should add 10.207/16 to our nets, should I?
'''
"Illegal RDNS-Domain: found no matching net for 207.10.in-addr.arpa (found nets: ['1.e.e.e.3.e.4.e.9.3.d.f.ip6.arpa', '169.10.in-addr.arpa'])"
'''
I'd still like to see the icvpn stuff resolved:
'''
wusel@ysabell:~$ traceroute srv01.ffhh
traceroute to srv01.ffhh (10.112.1.1), 30 hops max, 60 byte packets
 1  10.169.6.1 (10.169.6.1)  39.494 ms  40.054 ms  40.936 ms
 2  10.169.1.1 (10.169.1.1)  57.570 ms  58.001 ms  58.568 ms
 3  hamburg03.icvpn (10.207.0.63)  78.205 ms  79.013 ms  79.888 ms
 4  srv01.ffhh (10.112.1.1)  98.418 ms  98.953 ms  100.933 ms
'''
Suggestions?

**Comment by leeps** [comment]
Create a new file for ICVPN, add network and domains there.
Publish your script, too, to enable other communities to host additional name servers for redundancy :)

**Comment by mweinelt** [comment]
ack @leeps.

**Comment by wusel42** [comment]
Moved the stuff, code will be added to github soon; currently lookig at automating the v6-rev-zone.

File: mueritz [hunk 1]
```diff
@@ -12,7 +12,7 @@ bgp:
     ipv6: fec0::a:cf:0:8a
 domains:
   - mueritz
+  - icvpn
```
Comment (leeps) [inline mueritz]: This line still needs to go, right?

**Comment by wusel42** [comment]
Please reviewa again, should be merge-able now.

**Comment by wusel42** [comment]
Hmm, what does this mean?

Checking meta-icvpn
IPv4 Network overlap: meta-icvpn (10.207.0.0/16), transfer (10.207.0.0/16)
IPv6 Network overlap: meta-icvpn (fec0::a:cf:0:0/96), transfer (fec0::a:cf:0:0/96)

For this to work, "transfer" needs to be removed, which seems to be defined outside the repository.

**Comment by wusel42** [comment]
git suxx

**Pull request closed without merging**